deps.ts

```diff
@@ -21,7 +22,12 @@ export type {
   EventHandlers,
   Guild,
   Interaction,
+  InteractionDataOption,
+  InteractionResponse,
   MakeRequired,
   Message,
   User,
-} from "https://deno.land/x/discordeno@13.0.0-rc35/mod.ts";
+} from "https://deno.land/x/discordeno@13.0.0-rc42/mod.ts";
+
+export * as Oak from "https://deno.land/x/oak@v10.6.0/mod.ts";
```

Comment:
I want not to mix discordeno functions and oak functions for the readability.
So, I separated the oak libs into the `Oak` namespace, here.